R/super.R

```diff
@@ -36,20 +36,80 @@
 super <- function(...) {
   # TODO: (RK) Error on anonymous function calls.
   fn <- as.character(sys.call(-1)[[1]])
-
-  env <- parent.frame(2)
+  if (identical(fn[1], "get")) {
```

Comment:
The `get(fn, envir = super::multi_parent_env(parent.frame(level), parent_count)) # parent.frame(), parent_count))` call below translates to `c("get", fn, ...)`